README.md

```diff
@@ -1,115 +1,42 @@
-# go-policy-template
+# psp-hostpaths-policy
 
-This is a template repository that can be used to to quickly scaffold a
-Kubewarden policy written with Go language.
+Replacement for the Kubernetes Pod Security Policy that controls the usage of
+hostPath volumes.
```

Comment:
Can you add an additional line that says something like:

> The policy inspects both the containers and the init containers that are using `hostPath` volumes.

Reply:
True! done in https://github.com/kubewarden/psp-hostpaths-policy/pull/1/commits/ec7b3cc05b34447cf5e275baa28e646589299dc1.